java-frontend/src/test/files/se/MethodInvocationLeadingToNPE.java

```diff
@@ -62,8 +62,8 @@ static void npeIfArg0IsTrueAndArg1IsNull(boolean arg0, Object arg1, boolean arg2
     }
   }
 
-  static void nullable(@Nullable Object o) {
-    o.toString(); // Noncompliant {{A "NullPointerException" could be thrown; "o" is nullable here.}} flow@foo5 [[order=3]] {{'NullPointerException' is thrown.}}
+  static void nullable(@Nullable Object o) { // flow@foo5 [[order=3]] {{Implies 'o' can be null.}}
```

Comment:
This one is bothering me. The message is not coherent with what we just said in two previous flow messages:

- `implies 'o' is null.`
- `'o' is passed to nullable().`
- `implies 'o' can be null.` (it IS null in this flow)